examples/kaniko/skaffold.yaml

```diff
@@ -4,8 +4,9 @@ build:
   artifacts:
   - imageName: gcr.io/k8s-skaffold/skaffold-example
   kaniko:
-    gcsBucket: k8s-skaffold
-    pullSecret: ~/secrets/kaniko/kaniko.json
+    gcsBucket: skaffold-kaniko
+    pullSecretName: e2esecret
+    namespace: default
```

Comment:
I don't think we want to run the integration test in the default namespace

Reply:
It doesn't run there overall, just the kaniko build. The issue is that the secret needs to be pre-created in the cluster in a fixed location, I could put it in a different namespace if we want, but it can't be dynamically created.

Reply:
It could be dynamically created if we pointed to the secret json itself on the builder. We can merge this now, and if we run into trouble switch it to dynamically created.

Reply:
Yeah, we could definitely store the secret.json in kokoro and inject it into the builder as well.